versioned_docs/version-0.2.0/contribute-to-kubeslice-docs.md

```diff
@@ -0,0 +1,52 @@
+# Contribute to KubeSlice documentation
+
+_KubeSlice welcomes suggestions for improvement from all contributors, new and experienced!_
+
+You can contribute to the KubeSlice documentation by
+
+- Improving existing content
+- Suggesting new content
+
+## Getting Started
+
+Everybody is welcome to suggest changes by submitting a pull request
+or report a bug/typo with the help of a GitHub issue to the
+[docs](https://github.com/kubeslice/docs) repository.
+
+## Pre-requisites 
+
+Below is the list of things you should be aware of before getting started with your contribution journey to the KubeSlice documentation
```

Comment:
The sentence is a leading to the list that follows it. So, it should end with a colon `:`.